final-project/templates/index.html

```diff
@@ -7,56 +7,76 @@
 {% endblock %}
 
 {% block head %}
-
-    <!--COMMENTED OUT CODE FOR LATER: STOCK PURCHASE CONFIRMATION-->
-    <!--<head>-->
-    <!--    <div class="alert alert-info" role="alert">-->
-    <!--        Bought!-->
-    <!--    </div>-->
-
-    <!--</head>-->
-
+    HOMEPAGE
 {% endblock %}
 
 {% block main %}
 
-    <table class="table table-striped">
-        <thead>
-            <tr>
-                <th>Symbol</th>
-                <th>Shares</th>
-                <th>Price</th>
-                <th>TOTAL</th>
-            </tr>
-        </thead>
-        <tfoot>
-            <tr>
-                <td colspan="3"></td>
-                <td>{{total_assets}}</td>
-            </tr>
-        </tfoot>
-        <tbody>
-
-            {% for stock in range(count_stock) %}
-                <tr>
-                    <td>{{user_assets[stock]["symbol"]}}</td>
-                    <td>{{user_assets[stock]["shares"]}}</td>
-                    <td>{{current_price[stock]}}</td>
-                    <td>{{stock_ownership[stock]}}</td>
-                </tr>
-
-            {% endfor %}
-
-            <tr>
-                <td>CASH</td>
-                <td></td>
-                <td></td>
-                <td>{{cash_holdings}}</td>
-            </tr>
+    <div id=welcome>
+        <header>
+            <h1>Welcome back {{this_username}}!</h1>
+        </header>
+    </div>
+
+    <select class="nav-search-dropdown searchSelect" id="searchDropdownBox" name="url" title="Search in">
+        <option selected="selected" value="search-alias=aps">All Departments</option>
+        <option value="search-alias=appliances">Appliances</option>
```

Comment:
If this is a template, there is a good argument for making these options auto-generated based on the data you support

Reply:
Good point, I can copy the code from the category.html template, because I automated them there.

I need to start adding javascript and better css at some point as well, which will make these templates a little more dynamic and better looking. But first, I want to finish all the application.py functionality.

Reply:
<img width="1099" alt="screen shot 2017-12-10 at 6 58 35 am" src="https://user-images.githubusercontent.com/8930410/33804659-97b4fd36-dd77-11e7-9c9c-8c4196b010e0.png">

Hey, it looks like Amazon passes back a search alias rather than the actual value in the drop down (Oh, this list is from Amazon by the way). Do you think they do this to save space in their database? All the aliases do not have spaces, and they are shorter than the value in the drop-down. I wonder if I should learn something from this, but I am not sure what it would be. 

Anyway, I can search for online answers to this later. No need to spend time on it. Only thought it was interesting.